modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/io/PageIo.java

```diff
@@ -198,7 +198,7 @@ public static int getVersion(long pageAddr) {
      * @param pageAddr Page address.
      * @param ver Version.
      */
-    protected static void setVersion(long pageAddr, int ver) {
+    public static void setVersion(long pageAddr, int ver) {
```

Comment:
I'm sorry, why does it need to be public?

Reply:
For **FilePageStoreTest#createPageByteBuffer**